### PR TITLE
Add documentation for enum values

### PR DESCRIPTION
**Pull request by richardm-stripe**
Generated from `openapi 8ec28b9` in anticipation of #905 being merged first.

### CODE REVIEW

File: src/main/java/com/stripe/model/Account.java [hunk 1]
```diff
@@ -108,7 +111,10 @@
   @SerializedName("tos_acceptance")
   TosAcceptance tosAcceptance;
 
-  /** The Stripe account type. Can be `standard`, `express`, or `custom`. */
+  /**
+   * The Stripe account type. Can be `standard`, `express`, or `custom`. One of `custom`, `express`,
+   * or `standard`.
```
Comment (ob-stripe) [inline src/main/java/com/stripe/model/Account.java]: :/ Can you add "Can be" to the list of strings to look for?

File: src/main/java/com/stripe/model/Account.java [hunk 2]
```diff
@@ -598,25 +604,28 @@ public CapabilityCollection capabilities(AccountCapabilitiesParams params, Reque
   public static class Capabilities extends StripeObject {
     /**
      * The status of the card issuing capability of the account, or whether you can use Issuing to
-     * distribute funds on cards.
+     * distribute funds on cards One of `active`, `inactive`, or `pending`.
```
Comment (ob-stripe) [inline src/main/java/com/stripe/model/Account.java]: Looks like the original documentation string is lacking a final `.`. Another reason to prefix the enum string with `<p>`.

**Comment by richardm-stripe** [comment]
Fixed
r? @ob-stripe

**Comment by richardm-stripe** [comment]
@ob-stripe this is good to go again. Also re: https://git.corp.stripe.com/stripe-internal/pay-server/pull/189693#issuecomment-1119406

Turns out that the closing `</p>` tags are removed by `./gradle spotlessApply`. Apparently this is a standard Javadoc thing.

**Pull request merged**